### ISSUE TITLE
Build issue: Cannot find MLIRMemRefTransforms.lib

### ISSUE THREAD

**Issue by Edouard127** [llvm/llvm-project#61404]
Hi, I've recently tried to build LLVM for a project, but I'm unable to successfully complete the build process since It seems to be missing a file

Those are the only errors and warnings I've got while building

I have the Windows SDK 10.0.19041.0
Default VS Clang compiler for C and C++

```

Here are the errors

CMake Warning (dev) at C:/Program Files/CMake/share/cmake-3.24/Modules/GNUInstallDirs.cmake:243 (message):
  Unable to determine default CMAKE_INSTALL_LIBDIR directory because no    
  target architecture is known.  Please enable at least one language before
  including GNUInstallDirs.                                                
Call Stack (most recent call first):                                       
  C:/Users/Administrator/Desktop/BetterElytraBot/TypeScriptCompiler/3rdParty/llvm-project/llvm/cmake/modules/LLVMInstallSymlink.cmake:5 (include)
  tools/llvm-ar/cmake_install.cmake:48 (include)
  tools/cmake_install.cmake:39 (include)
  cmake_install.cmake:69 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Error at tools/mlir/lib/Dialect/MemRef/Transforms/cmake_install.cmake:37 (file):
  file INSTALL cannot find
  "C:/Users/Administrator/Desktop/BetterElytraBot/TypeScriptCompiler/__build/llvm/debug/lib/MLIRMemRefTransforms.lib":
  No such file or directory.
Call Stack (most recent call first):
  tools/mlir/lib/Dialect/MemRef/cmake_install.cmake:38 (include)
  tools/mlir/lib/Dialect/cmake_install.cmake:66 (include)
  tools/mlir/lib/cmake_install.cmake:40 (include)
  tools/mlir/cmake_install.cmake:55 (include)
  tools/cmake_install.cmake:45 (include)
  cmake_install.cmake:69 (include)
```

And that is the command
`cmake ..\..\3rdParty\llvm-project\llvm -G "Visual Studio 16 2019" -A x64 -DLLVM_TARGETS_TO_BUILD="host" -DLLVM_EXPERIMENTAL_TARGETS_TO_BUILD=WebAssembly -DCMAKE_BUILD_TYPE=Debug -Thost=x64 -DCMAKE_INSTALL_PREFIX=../../3rdParty/llvm/debug -DLLVM_INSTALL_UTILS=ON -DLLVM_ENABLE_ASSERTIONS=ON -DLLVM_ENABLE_PLUGINS=ON -DLLVM_ENABLE_PROJECTS="clang;lld;mlir" -DLLVM_ENABLE_EH=ON -DLLVM_ENABLE_RTTI=ON -DLLVM_REQUIRES_RTTI=ON -DLLVM_ENABLE_PIC=ON`

Then
`cmake --build . --config Debug --target install -j 8
cmake --install . --config Debug`

I hope anyone can help me
